app/src/main/java/com/google/samples/apps/sunflower/workers/SeedDatabaseWorker.kt

```diff
@@ -27,25 +27,29 @@ import com.google.samples.apps.sunflower.data.AppDatabase
 import com.google.samples.apps.sunflower.data.Plant
 import com.google.samples.apps.sunflower.utilities.PLANT_DATA_FILENAME
 
-class SeedDatabaseWorker(context: Context, workerParams: WorkerParameters) : Worker(context, workerParams) {
+class SeedDatabaseWorker(
+    context: Context,
+    workerParams: WorkerParameters
+) : Worker(context.applicationContext, workerParams) {
```

Comment:
Sorry I missed this the last review round.  Can you change `context.applicationContext` to just `context`?

Reply:
I prefer to use `context.applicationContext`, because, internal there is hard-reference to a `Context` as I know,  if the user passes `Activity`  to `SeedDatabaseWorker`, there would be leak problem.
In order to keep this class pure general(generic), I'd like to pass non-activity reference to parent.

Reply:
@tiembo Pls take a look here `androidx.work.ListenableWorker` which is the parent of  `Worker` .